src/Security/Security/Az.Security.psd1

```diff
@@ -12,7 +12,7 @@
 # RootModule = ''
 
 # Version number of this module.
-ModuleVersion = '0.7.8'
+ModuleVersion = '0.8.0'
```

Comment:
no need to change the version, it will be updated by azure-powershell team each release. Please revert it back